src/tracking.js

```diff
@@ -1,9 +1 @@
-import "autotrack/lib/plugins/url-change-tracker";
-
-const googleScript = document.createElement("script");
-googleScript.setAttribute("async", true);
-googleScript.setAttribute(
-  "src",
-  "https://www.google-analytics.com/analytics.js"
-);
-document.head.appendChild(googleScript);
+// Paste your tracking code (e.g. Google Analytics) here.
```

Comment:
* [ ] As we introduced this script solely for GA (#32) I think it's cleaner to drop it now.

Reply:
If we drop it there is also no need for the cookie consent. But I would like to keep the cookie consent code around, so that it is fairly easy to use it.

Reply:
How about applying the consent at some existing point, like Gmaps or videos, where we currently always use the no-cookies version?

FYI The red flag for me here was the comment plus the fact that the GA code did not (only) live here, which may be misleading.